### PR TITLE
Add <Table> type definitions

### PR DESCRIPTION
**Pull request by giladsegal**

### CODE REVIEW

File: packages/wsr-types/src/components/DataTable.d.ts [hunk 1]
```diff
@@ -21,6 +14,7 @@ declare namespace __WSR {
       onRowClick?: (rowData: any, rowNum: number) => void;
       onMouseEnterRow?: (rowData: any, rowNum: number) => void;
       onMouseLeaveRow?: (rowData: any, rowNum: number) => void;
+      onSortClick?: (column: DataTableColumn, colNum: number) => void;
```
Comment (giladsegal) [inline packages/wsr-types/src/components/DataTable.d.ts]: this prop is missing in WSR in `propTypes`, can you add it?

File: packages/wsr-types/src/drivers/DropdownLayout.driver.d.ts [hunk 2]
```diff
@@ -23,7 +23,7 @@ declare namespace __WSRTests {
     mouseLeave: () => void;
     mouseLeaveAtOption: (position: number) => void;
     /** @deprecated Use optionDriver*/
-    optionAt: (position: number) => Node;
+    optionAt: (position: number) => HTMLElement;
```
Comment (giladsegal) [inline packages/wsr-types/src/drivers/DropdownLayout.driver.d.ts]: can you go over
https://wix.slack.com/archives/G7KTULSNS/p1574149786001500
and fix driver in WSR?

File: packages/wsr-types/src/components/Table.d.ts [hunk 3]
```diff
@@ -0,0 +1,98 @@
...
+    const EmptyState: React.SFC<EmptyState.EmptyStateProps>;
+    const BulkSelectionCheckbox: React.SFC<{ dataHook: string }>;
+
+    export type TableColumn = DataTable.DataTableColumn & {
+      style?: string;
+      width?: string;
+      important?: boolean;
+    };
+
+    export type TableColumnAlign = "start" | "center" | "end";
```
Comment (mosheKerbel) [inline packages/wsr-types/src/components/Table.d.ts]: looks redundant

File: packages/wsr-types/src/components/DataTable.d.ts (hunk 1 above)
Comment (mosheKerbel) [reply packages/wsr-types/src/components/DataTable.d.ts]: https://github.com/wix/wix-style-react/pull/4650

File: packages/wsr-types/src/drivers/DropdownLayout.driver.d.ts (hunk 2 above)
Comment (mosheKerbel) [reply packages/wsr-types/src/drivers/DropdownLayout.driver.d.ts]: I wanted to test it but i'm having some unrelated issue with running the tests on my localhost
not sure whether this change is risky or not

Comment (mosheKerbel) [reply packages/wsr-types/src/drivers/DropdownLayout.driver.d.ts]: anyways, besides that, I went over the PR and it looks great

Comment (giladsegal) [reply packages/wsr-types/src/drivers/DropdownLayout.driver.d.ts]: I will remove it from the PR for this time

**Pull request merged**